bot/exts/easter/earth_photos.py

```diff
@@ -0,0 +1,46 @@
+import discord
+from discord.ext import commands
+
+from bot.constants import Colours
+from bot.constants import Tokens
+
+UnClient_id = Tokens.unsplash_key
+
+
+class EarthPhotos(commands.Cog):
+    """This cog contains the command for earth photos."""
+
+    def __init__(self, bot: commands.Bot):
+        self.bot = bot
+
+    @commands.command(aliases=["earth"])
+    async def earth_photos(self, ctx: commands.Context) -> None:
+        """Returns a random photo of earth, sourced from Unsplash."""
+        async with ctx.typing():
+            async with self.bot.http_session.get(
+                    'https://api.unsplash.com/photos/random',
+                    params={"query": "earth", "client_id": UnClient_id}) as r:
+                jsondata = await r.json()
+                linksdata = jsondata.get("urls")
+                embedlink = linksdata.get("regular")
+                downloadlinksdata = jsondata.get("links")
+                userdata = jsondata.get("user")
+                username = userdata.get("name")
+                userlinks = userdata.get("links")
+                profile = userlinks.get("html")
+            async with self.bot.http_session.get(
+                downloadlinksdata.get("download_location"),
+                    params={"client_id": UnClient_id}) as er:
+                er.status
```

Comment:
Uhhh what does line do?

Reply:
https://github.com/python-discord/sir-lancebot/pull/609#pullrequestreview-602548202

Reply:
Sorry I didn’t get you.

Reply:
https://help.unsplash.com/en/articles/2511245-unsplash-api-guidelines

Reply:
Am talking about `er.status`.

Reply:
I have to call `er.status` otherwise the lint fails

Reply:
because then flake8 returns `er is defined but never used`

Reply:
Flake8 never fails on this issue. You have added it in b2ee1b9e79766b8cfe7b45670f6c86e7166ee84f commit and I don’t see it failing before. And you can replace it with `_`, it is used to indicate that the  result is being deliberately ignored.

Reply:
oh sorry, i ran flake8 locally and it freaked on me

Reply:
how do you mean?